SafeguardSessions/CHANGELOG.md

```diff
@@ -2,6 +2,29 @@
 
 ## [Unreleased]
 
+### Compatible SPS Versions
+
+* 7.3.0, 7.4.0
+
+### Breaking changes
+
+* Power BI Service support has been implemented and the order and type of the input parameters have changed. After the upgrade, the input parameters of previously saved .pbix files must be transformed accordingly to work with the new versions of One Identity Safeguard Power BI Connector and One Identity Safeguard Power BI Report Template.<br>
+*Transformation*: In Power BI Desktop, open **Transform Data**, select **Source**, and overwrite the parameter list of *SafeguardSessions.Contents*.

```

Comment:
In DM we have talked about writing the customer is advised to check it current connector's input parameters before upgrading and overwriting it. Can you please add it note|tip?

Reply:
Even with the table?

Reply:
Done.

Reply:
Actually it is not. Can you write an sentence explicitly stating that we advise the customer the check the input parameter before the updgrade?

In DM we have talked about something like this:
> Before upgrading the Power BI Connector, we recommend to check what input parameters were used when connecting to your SPS so that after replacing the old version of the connector with the new one, you know how to fix the order and type of the input parameters.

We have also agreed that we won't reference Power BI desktop UI layout that may change in the future, so can you remove Transform Data and its related things?

Reply:
This is the latest commit.
https://github.com/OneIdentity/SafeguardPowerBI/compare/cdba570673acc1f159377596ef60cb8e448a2fc2..2585b6e2de3f0deb3868ddcb593c75a1b178ba37

We also noted that transform data is not likely to change. But I'll fix that.

Reply:
The changelog has been updated. I used the official documentation to navigate the user to specific steps. I also used the official name of the layout elements instead of buttons.